scripts/generate_catalog/generate_catalog.go

```diff
@@ -56,6 +56,7 @@ var keywordsPool = []string{
 
 var (
 	branchesToSkip = []string{
+		"set-name-prefix/v0.0",
```

Comment:
I think this works as a stopgap, but ideally we should maintain a consistent pattern/UX across functions IMO. If the goal is to migrate to a new convention, we should still make sure things are consistent (and not broken) in the transition

Reply:
Completely agree. I'll sync with the team after the holidays to confirm the path forward for this function in regards to the documentation.